File: app/src/main/java/life/plank/juna/zone/data/network/model/LiveTimeStatus.java
package life.plank.juna.zone.data.network.model;

import lombok.Data;

@Data
public class LiveTimeStatus {
    private String timeStatus;
    private int minute;
    private int extraMinute;

    public LiveTimeStatus(String timeStatus, int minute, int extraMinute) {
        this.timeStatus = timeStatus;
        this.minute = minute;
        this.extraMinute = extraMinute;
    }
}

File: app/src/main/java/life/plank/juna/zone/data/model/LiveTimeStatus.kt
package life.plank.juna.zone.data.model

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class LiveTimeStatus(
        var timeStatus: String = "",
        var minute: Int = 0,
        var extraMinute: Int = 0
) : Parcelable